web/source/dom/touchAliasElement.ts

```diff
@@ -30,26 +30,83 @@ namespace com.keyman.dom {
+        }
+      };
+
+      // Very rough breadcrumbs for use with any Sentry logs.
+
+      // We'd expect to see the telltale 'keymanweb-input' appear as the class for one of the
+      // three elements logged here... at least, normally.
+      console.info(`(For diagnosing https://github.com/keymanapp/keyman/issues/4797)`);
+      // We don't use .innerHTML because that would tell us what the user's been typing.
+      // No keylogging!
+      console.info(`Original target: ${elementDescriber(target)}`);
+      if(target.previousSibling && target.previousSibling instanceof HTMLElement) {
+        console.info(`Previous sibling: ${elementDescriber(target.previousSibling)}`);
+      }
+      if(target.nextSibling && target.nextSibling instanceof HTMLElement) {
+        console.info(`Next sibling: ${elementDescriber(target.nextSibling)}`);
+      }
+      console.info(`Target's parent: ${elementDescriber(target.parentElement)}`);
+      if(target.parentElement instanceof HTMLElement) {
+        console.info(`Target's grandparent: ${elementDescriber(target.parentElement.parentElement)}`);
+      }
+      // Triggers the actual Sentry log (on pages that include Sentry logging)
+      throw new Error(`Unexpected touch-start event target.`);
+    };
+
     // Identify the scroller element
     if(target && dom.Utils.instanceof(target, "HTMLSpanElement")) {
       scroller=target.parentNode as HTMLElement;
     } else if(target && (target.className != null && target.className.indexOf('keymanweb-input') >= 0)) {
       scroller=target.firstChild as HTMLElement;
     } else if(target && dom.Utils.instanceof(target, "HTMLDivElement")) {
-      // Two possibilities:  the scroller & the blinking DIV of the caret.
+      // Three possibilities:  the scroller, the scrollbar, & the blinking DIV of the caret.
       // A direct click CAN trigger events on the blinking element itself if well-timed.
       scroller=target;
 
       // Ensures we land on the scroller, not the caret.
       if(scroller.parentElement && scroller.parentElement.className.indexOf('keymanweb-input') < 0) {
         scroller = scroller.parentElement;
       }
+
+      // scroller is now either the actual scroller or the scrollbar element.
+      // We don't return either of these, and they both have the same parent element.
     } else if(target['kmw_ip']) { // In case it's called on a TouchAliasElement's base (aliased) element.
       return target['kmw_ip'] as TouchAliasElement;
     } else {
       // If it's not in any way related to a TouchAliasElement, simply return null.
+      // NOTE:  At present, this method should only be called in places that actually EXPECT a TouchAliasElement...
+      // So if we fail to find one, something's gone wrong, and we'd like more details.
+
+      // There's an evasive error related to this at present so we'll temporarily console-log info 
+      // that may help us determine why we couldn't find the expected element.
+      elementLogger();
```

Comment:
So, where before we were returning `null` for the unexpected 'fallback' case (which, again, leads to the `setTextCaret` - `undefined` error), we're now logging + throwing errors to trigger Sentry log capture on keymanweb.com with something more informative.